trace-shield/helm/trace-shield/templates/keto-configmap.yaml

```diff
@@ -56,16 +56,97 @@ data:
     class ObservabilityTenant implements Namespace {
       related: {
         organizations: Organization[]
-        viewers: (User | SubjectSet<Group, 'members'> | OAuth2Client)[]
-        editors: (User | SubjectSet<Group, 'members'>)[]
+
```

Comment:
Should this be refactored to an individual file that's templated in?

Reply:
Possibly, but it might also be possible to fetch it from a URL for the given release from GitHub or something.